modules/apps/portal-template/portal-template-react-renderer-api/src/main/java/com/liferay/portal/template/react/renderer/ComponentDescriptor.java

```diff
@@ -67,6 +86,8 @@ public boolean isPositionInLine() {
 		return _positionInLine;
 	}
 
+	private final String _additionalProps;
```

Comment:
This being a String means you can only pass 1 file for `additionalProps`. I considered making this a Collection, but as @wincent likes to say... [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)